test/system/001-basic.bats

```diff
@@ -190,7 +190,9 @@ See 'podman version --help'" "podman version --remote"
     run_podman --log-level=error   info
     run_podman --log-level=fatal   info
     run_podman --log-level=panic   info
+    # docker compat
     run_podman --debug   info
+    run_podman -D        info
```

Comment:
LGTM. If for any reason you need to re-push (which I see no reason for), something like this would be welcome:
```bash
assert "$output" =~ " level=debug "
assert "$output" =~ "Initializing boltdb state at" "confirm that -D turns on debug msgs"
```
Reason: I always like to confirm effects.

I'm working on a system test cleanup PR, and will wait for this to merge so I can include it in my PR. Am commenting simply as a reminder for next time.